dev/AnimatedVisualPlayer/AnimatedVisualPlayer.cpp

```diff
@@ -393,12 +405,14 @@ void AnimatedVisualPlayer::OnUnhiding()
     }
 }
 
+// Public API.
```

Comment:
Are the public API comments useful?

Reply:
They save the reader from having the read the header file in order to understand that these are public. The problem they solve for me is to make is easier to reason about reentrance. At each DP setter I need to consider that code could reenter from any of the public methods. Being able to easily identify the public methods makes it faster to do that reasoning.

---
In reply to: [343380321](https://github.com/microsoft/microsoft-ui-xaml/pull/1558#discussion_r343380321) [](ancestors = 343380321)